.github/actions/ci/action.yml

```diff
@@ -82,4 +82,4 @@ runs:
       shell: bash
       # Add a brief sleep here to ensure the test service is ready to receive
       # requests
-      run: sleep 5 && make run-contract-tests
```

Comment:
You could use the new argument I added to the sdk-test-harness

`-status-timeout 15` or something.